en_US/webapp/src/main/webapp/i18n/all_en_US.properties

```diff
@@ -981,3 +981,31 @@ file_upload_predicate = predicate
+acct_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Congratulations!</strong>\n </p>\n\n <p>\n We have created your new account on ${siteName}, associated with ${userAccount.emailAddress}.\n </p>\n\n <p>\n If you did not request this new account you can safely ignore this email.\n This request will expire if not acted upon for 30 days.\n </p>\n\n <p>\n Click the link below to create your password for your new account using our secure server.\n </p>\n\n <p>\n <a href="${passwordLink}" title="password">${passwordLink}</a>\n </p>\n\n <p>\n If the link above doesn't work, you can copy and paste the link directly into your browser's address bar.\n </p>\n\n <p>\n Thanks!\n </p>
+
+password_reset_pending_email_html_text = <p>\n Dear ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n We have received a request to reset the password for your ${siteName} account (${userAccount.emailAddress}).\n </p>\n\n <p>\n Please follow the instructions below to proceed with your password reset.\n </p>\n\n <p>\n If you did not request this new account you can safely ignore this email.\n This request will expire if not acted upon within 30 days.\n </p>\n\n <p>\n Click on the link below or paste it into your browser's address bar to reset your password\n using our secure server.\n </p>\n\n <p><a href="${passwordLink}" title="password">${passwordLink}</a> </p>\n\n <p>Thank you!</p>
+password_reset_pending_email_plain_text = Dear ${userAccount.firstName} ${userAccount.lastName}:\n\nWe have received a request to reset the password for your ${siteName} account\n(${userAccount.emailAddress}).\n\nPlease follow the instructions below to proceed with your password reset.\n\nIf you did not request this new account you can safely ignore this email.\nThis request will expire if not acted upon within 30 days.\n\nPaste the link below into your browser's address bar to reset your password\nusing our secure server.\n\n${passwordLink}\n\nThank you!\n
+password_reset_pending_email_subject = ${siteName} reset password request
+
+password_reset_complete_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Password successfully changed.</strong>\n </p>\n\n <p>\n Your new password associated with ${userAccount.emailAddress} has been changed.\n </p>\n\n <p>\n Thank you.\n </p>
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPassword successfully changed.\n\nYour new password associated with ${userAccount.emailAddress}\nhas been changed
+
+confirm_email_changed_email_html_text = <p>\n Hi, ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n You recently changed the email address associated with\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Thank you.\n </p>
```

Comment:
I would move comma (,) after the last name:
Hi Dragan Ivanovic,

Actually, I think we should use here Dear , not Hi (as line 991). If we don't want to risk with language practices for using comma, we might also adopt as in line 991

Dear Dragan Ivanovic:

Reply:
Fixed